velox/functions/lib/ArraySort.cpp

```diff
@@ -495,9 +505,36 @@ core::CallTypedExprPtr asArraySortCall(
 
 } // namespace
 
+std::shared_ptr<exec::VectorFunction> makeArraySortLambdaFunction(
+    const std::string& /*name*/,
+    const std::vector<exec::VectorFunctionArg>& inputArgs,
+    const core::QueryConfig& /*config*/,
+    bool ascending,
+    bool throwOnNestedNull) {
+  VELOX_CHECK_EQ(inputArgs.size(), 2);
+  return std::make_shared<ArraySortLambdaFunction>(
+      ascending, throwOnNestedNull);
+}
+
+std::shared_ptr<exec::VectorFunction> makeArraySort(
+    const std::string& /*name*/,
+    const std::vector<exec::VectorFunctionArg>& inputArgs,
+    const core::QueryConfig& /*config*/,
+    bool ascending,
+    bool nullsFirst,
+    bool throwOnNestedNull) {
+  return create(inputArgs, ascending, nullsFirst, throwOnNestedNull);
```

Comment:
Do we need this function as it just calls another function? Perhaps moves the logic of create into makeArraySort and remove create function.